spec.html

```diff
@@ -28728,29 +28517,12 @@ <h1>Math.random ( )</h1>
       <emu-clause id="sec-math.round">
         <h1>Math.round ( _x_ )</h1>
         <p>Returns the Number value that is closest to _x_ and is an integer. If two integers are equally close to _x_, then the result is the Number value that is closer to *+&infin;*. If _x_ is already an integer, the result is _x_.</p>
-        <ul>
-          <li>
-            If _x_ is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is *+0*, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is *-0*, the result is *-0*.
-          </li>
-          <li>
-            If _x_ is *+&infin;*, the result is *+&infin;*.
-          </li>
-          <li>
-            If _x_ is *-&infin;*, the result is *-&infin;*.
-          </li>
-          <li>
-            If _x_ is greater than 0 but less than 0.5, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is less than 0 but greater than or equal to -0.5, the result is *-0*.
-          </li>
-        </ul>
+        <p>When the `Math.round` method is called with one argument _x_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _n_ be ? ToNumber(_x_).
+          1. If _n_ is an integral Number, return _n_.
+          1. Return the integral Number closest to _n_, preferring the Number closer to *+&infin;* in the case of a tie.
```

Comment:
it seems important not to lose the explicit mention of `0.5` here as part of the rounding method.

Reply:
Do you have a concrete suggestion? I personally think the current text is sufficient.

Reply:
```suggestion
          1. If _x_ &gt; 0 and _x_ &lt; 0.5, return *+0*.
          1. If _x_ &lt; 0 and _x_ &ge; -0.5, return *-0*.
          1. Return the integral Number closest to _n_, preferring the Number closer to *+&infin;* in the case of a tie.
```

Reply:
Either is fine by me.